bigcode_eval/tasks/humaneval.py

```diff
@@ -56,25 +63,57 @@ def __init__(self, strip_prompt, k=[1, 10, 100], num_workers=16, timeout=3.0):
         self.k = k
         self.num_workers = num_workers
         self.timeout = timeout
+        self.doc = None
+        self.nuggets_config = nuggets_config
 
     def get_dataset(self):
         """Returns dataset for the task or an iterable of any object, that get_prompt can handle"""
         return self.dataset["test"]
 
-    def get_prompt(self, doc):
-        """Builds the prompt for the LM to generate from."""
+    def get_start_context(self):
+        return "Implement solutions to the following coding tasks given the function heading:\n"
+
+    def get_one_shot_example(self):
```

Comment:
Let's follow the guide and move this logic to `fewshot_examples()`. You can see some examples by searching through the codebase. Basically I would like to make this change while respecting as many bigcode conventions as possible to minimize the chance of mistakes.